libs/gf/configuration-danish-locale/project.json

```diff
@@ -1,6 +1,5 @@
 {
   "projectType": "library",
-  "root": "libs/gf/configuration-danish-locale",
```

Comment:
question: do you remember changing this line? It doesn't look like it's something that should be part of this PR. Can be reverted.

Reply:
Thx. Removed.